slash/core/fixtures/parameters.py

```diff
@@ -174,11 +178,18 @@ def as_transform(self, transform):
 
 class ParametrizationValue(object):
 
-    def __init__(self, label, value=NOTHING):
+    def __init__(self, label=NOTHING, value=NOTHING, tags=[]):
```

Comment:
Avoid passing a list as a default parameter - it should default to either `None` or `()`